client/my-sites/email/email-management/home/email-plan.jsx

```diff
@@ -267,13 +266,17 @@ class EmailPlan extends React.Component {
 	}
 
 	renderAddNewMailboxesOrRenewNavItem() {
-		const { domain, hasSubscription, purchase, selectedSite, translate } = this.props;
+		const { domain, purchase, translate } = this.props;
 
-		if ( ! domain.currentUserCanManage || ! hasSubscription ) {
-			return null;
+		if ( hasEmailForwards( domain ) ) {
+			return (
+				<VerticalNavItem { ...this.getAddMailboxProps() }>
+					{ translate( 'Add new email forwards' ) }
+				</VerticalNavItem>
+			);
 		}
 
-		if ( ! selectedSite || ! purchase ) {
+		if ( ! purchase ) {
```

Comment:
Did you check with an MVP account? I think no subscription also means no purchase, hence the placeholder may be there forever.

Reply:
No I didn't, I don't have such account.